Sandbox/Sandbox/RedirectionViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  RedirectionViewController.swift
+//  Sandbox
+//
```

Comment:
Could explain a little why this controller is needed and how/where it is used.

Reply:
Currently it is not used, but we will need it when we the user connect with Fido 2, because the login method returns us a TKN so we have to exchange it with an access token.
We build the url based on the tkn (oauth / authorize), then we send this url to this Class (RedirectionViewController) which will retrieve the CODE (The authorization code received from the initial authorization call) and subsequently trigger the authWithCode method (/ oauth / token) and at the end access the user's profile